Source/Core/loadArrayBuffer.js

```diff
@@ -28,12 +28,13 @@ define([
      * @see {@link http://www.w3.org/TR/cors/|Cross-Origin Resource Sharing}
      * @see {@link http://wiki.commonjs.org/wiki/Promises/A|CommonJS Promises/A}
      */
-    function loadArrayBuffer(url, headers, request) {
+    function loadArrayBuffer(url, headers, request, requestOptions) {
```

Comment:
`request` is meant for internal use only, so we can't actually add new public options after it (so this might force us to switch to an options object sooner rather than later.